src/skills-builder/skills-builder-steps/view-results/data/service.js

```diff
@@ -22,6 +23,21 @@ export async function getRecommendations(jobSearchIndex, productSearchIndex, car
 
       // add a new key to the recommendations object and set the value to the response
       data.recommendations[productType] = response;
+
+      // Get the list of skills for this job that intersect with the skills for the products returned
+      const productSkillsList = {};
+      response.forEach(product => {
+        const skills = product?.skills;
+        if (skills) {
+          skills.forEach((skill) => {
+            const jobSkill = skill?.skill;
+            if (jobSkill) {
+              productSkillsList[jobSkill] = (productSkillsList[jobSkill] || 0) + 1;
```

Comment:
Added a comment:
              ```
// Upsert an entry with the skill name and update the number
// of times the skill is found in the results
```
This would enable us to do some ad-hoc sorting of skills by how often they show up across all products in all product lines in the results. It could be useful to highlight a particularly hot skill and bubble it to the top of the skills list, for example.